README.md

```diff
@@ -2,9 +2,9 @@
 
 # How to migrate from React.js
 
-Due to recent news, you might consider moving away from React.js or any technology under Facebook’s BSD+Patents License. If you are considering migrating or if you're trying to evaluate the scope of a migration, we've provided you a table of contents to determine what you need to know.
+Due to recent news, you might be considering moving away from React.js or any technology under Facebook’s BSD+Patents License. If you are migrating or trying to evaluate the scope of a migration, we've provided you with a quick guide to help you make your decision.
```

Comment:
@gdi2290 @Mayho
`you might be considering moving` sounds weird as we are placing 2 adjectives next to one another. I like the original here. But keep the update to the second sentence as I did the same thing.